dapr/proto/runtime/v1/dapr.proto

```diff
@@ -28,6 +28,9 @@ service Dapr {
 
   // Deletes the state for a specific key.
   rpc DeleteState(DeleteStateRequest) returns (google.protobuf.Empty) {}
+  
+  // Perform transactions for a specified store

```

Comment:
nit: please keep verb consistent with the actual method. e.g. `Executes transactions for a specified store`

Reply:
Changed comments